igvc_rviz_plugins/configs/jessii_conf.rviz

```diff
@@ -8,11 +8,13 @@ Panels:
         - /Status1
```

Comment:
Some of these changes are arbitrary and should not be included.

Reply:
Specifically the one that gives the rotation of the camera

Reply:
I replaced the current config with the one from jessi, which is better organized and has a comprehensive list of the topics